.github/workflows/deltaengine.yaml

```
@@ -0,0 +1,160 @@
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_QA }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      # Download Api artifact
+      - name: Download API artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: asset-artifact
+      
+      # push api artifact to s3
+      - name: copy API jars to S3 
+        run: |
```

actionlint

property "extract_version" is not defined in object type {assetbuild: {outputs: {}; result: string}}

(expression)